doc/operator-guide/schema-migration-process.rst

```diff
@@ -2,8 +2,118 @@
 Schema Migration Process
```

Comment:
Bullets also overused in this section. I would remove most of them and just write it as prose.